aspnetcore/web-api/advanced/custom-formatters.md

```diff
@@ -18,7 +18,7 @@ ASP.NET Core MVC has built-in support for data exchange in web APIs by using JSO
 
 Use a custom formatter when you want the [content negotiation](xref:web-api/advanced/formatting#content-negotiation) process to support a content type that isn't supported by the built-in formatters (JSON and XML).
 
-For example, if some of the clients for your web API can handle the [Protobuf](https://github.com/google/protobuf) format, you might want to use Protobuf with those clients because it's more efficient. Or you might want your web API to send contact names and addresses in [vCard](https://wikipedia.org/wiki/VCard) format, a commonly used format for exchanging contact data. The sample app provided with this article implements a simple vCard formatter.
+For example, if some of the clients for your web API can handle the [Protobuf](https://github.com/google/protobuf) format, you might want to [use Protobuf](xref:grpc/index) with those clients because it's more efficient. Or you might want your web API to send contact names and addresses in [vCard](https://wikipedia.org/wiki/VCard) format, a commonly used format for exchanging contact data. The sample app provided with this article implements a simple vCard formatter.
```

Comment:
I personally wouldn't add this link. Using gRPC is a very different way of building services than web API. Anyone coming from the following doc might be confused: https://docs.microsoft.com/aspnet/core/grpc/comparison?view=aspnetcore-3.0.